res/controllers/midi-controller-api.d.ts

```diff
@@ -1,4 +1,6 @@
 
+type MidiInputHandler = (channel: number, control: number, value:number, status:number, group:string) => void;
+
```

Comment:
I can't spot the place, where this type is defined in our API. Is this a mapping specific type, that doesn't belong to the API?

Reply:
Its the type our input handlers are expected to be https://github.com/mixxxdj/mixxx/wiki/midi%20scripting#midi-input-handling-functions